spinalcordtoolbox/labels.py

```diff
@@ -267,6 +267,33 @@ def label_vertebrae(img: Image, vertebral_levels: Sequence[int] = None) -> Image
     return out
 
 
+def find_missing_label(img, ref):
```

Comment:
```suggestion
def check_missing_label(img, ref):
```

What do you think of this function name instead? In the usage for `MSE`, we don't do anything with the missing labels... we only use this function to warn the user. So, `check` might be a more descriptive word to use.